kotlinx-coroutines-core/common/src/flow/operators/Lint.kt

```diff
@@ -144,9 +144,18 @@ public inline fun <T> SharedFlow<T>.retryWhen(noinline predicate: suspend FlowCo
     level = DeprecationLevel.WARNING
 )
 @InlineOnly
-public suspend inline fun <T> SharedFlow<T>.toList(destination: MutableList<T> = ArrayList()): List<T> =
+public suspend inline fun <T> SharedFlow<T>.toList(): List<T> =
     (this as Flow<T>).toList()
 
+/**
+ * @suppress
+ */
+@InlineOnly
+public suspend inline fun <T> SharedFlow<T>.toList(destination: MutableList<T>): Nothing {
```

Comment:
Should this one be in lint file? 
Would be nice to have a test from the issue as well, as the reference for the future

Reply:
Not sure, but I guess it *is* for linting. It works the same as `toList`, but will warn about unreachable code below.

Reply:
True.
I am rather concerned that [in coroutines] `Lint` is something "don't use this, rather use that", though this works for me as well.

It's weird though that public declaration that we know people use is marked with `@suppress` and has no documentation